src/components/FlameGraph/FlameGraph.jsx

```diff
@@ -171,19 +180,39 @@ class FlameGraph extends Component {
         this.props.history.push({search: params.toString(),});
     }
 
+    handleLayoutChange(event, data) {
+        this.setState({layout: data.value}, () => {
+            this.state.chart
+                .inverted(this.state.layout == 'icicle')
+                .resetZoom()
+            localStorage.setItem('layout', this.state.layout)
+        })
+    }
+
     render() {
         const searchButton = 
         <Button color='red' size='small' onClick={this.handleSearchClick}>
             <Button.Content>Search</Button.Content>
         </Button>
-        
+        const layoutOptions = [
+            {
+                text: "Flame Layout",
+                value: "flame"
+            },
+            {
+                text: "Icicle Layout",
+                value: "icicle"
+            }
+        ]
+
         return (
             <div>
                 <Dimmer page inverted active={this.state.loading}>
                     <Loader size='huge' inverted>Loading</Loader>
                 </Dimmer> 
                 <Container style={styles.container}>
                     <Container textAlign='right'>
+                        <Dropdown selection options={layoutOptions} onChange={this.handleLayoutChange} defaultValue={this.state.layout} />
```

Comment:
Style is Ok. I would just reduce the size of the dropdown and add a space between it and the "Reset Zoom" button, like the other container items.